### ISSUE TITLE
DaaS REST API query does not take fully qualified container name, only simple name

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#1563]
For instance, the command:
```
curl -k -XGET https://localhost:8081/data/com.ligadata.kamanja.samples.containers.coughcodes/?icd9code=1
```
Results in:
```
The requested resource could not be found.
```
but the command:
```
curl -k -XGET https://localhost:8081/data/coughcodes/?icd9code=1
```
results in
```
{
	"id": "69110994-e436-4100-8a9d-a2ee0dafe51f",
	"status": "success",
	"statusCode": "1000",
	"statusDescription": "",
	"resultCount": 0,
	"result": []
}
```

**Comment by ligadata-sridhar** [comment]
Changed to accept fully qualified container and also short name in the URLs.

Fixed on branch 1.5.3-phoenix-daas-api.

**Comment by ligadata-pokuri** [comment]
Hi @ligadata-william,

I moved sridhar fix to 1.5.3-phoenix-daas-api-input-md-move branch. 1.5.3-phoenix-daas-api-input-md-move is super set of 1.5.3-phoenix-daas-api & 1.5.3-phoenix-daas-api-input.

Hi @ligadata-yousef , if you are doing any changes, please create another branch on top of 1.5.3-phoenix-daas-api-input-md-move. Don't use 1.5.3-phoenix-daas-api-input any more.

@ligadata-sridhar & @ligadata-krishna decided to support aliases instead of full name. Now @ligadata-sridhar is supporting both short name or long name.

**Comment by ligadata-william** [comment]
@ligadata-sridhar 
This issue is resolved and I will close it but another issue has come in its place: #1564
